models/public/alexnet/model.yml

```diff
@@ -46,4 +46,5 @@ model_optimizer_args:
   - --input_model=$dl_dir/alexnet.caffemodel
   - --input_proto=$dl_dir/alexnet.prototxt
 framework: caffe
+quantizable: yes
```

Comment:
@IRDonch, how many models should be `quantizable`? Could we set `yes` as default?

Reply:
I got the current list from @vladimir-dudnik. We may extend it later as we determine that they still have acceptable quality after quantizing.

>  Could we set yes as default?

No, because we can't trust that an arbitrary model will still have good quality after quantizing (or that it will quantize at all).

Reply:
Ok, thanks!